index.html

```diff
@@ -1,48 +1,78 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="style.css">
-    <title>Awesome Books</title>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Awesome Books</title>
 </head>
 <body>
-  <h1>Awesome books</h1> <br> <br>
-  <table class="table table-striped">
-    <thead>
-      <tr class="t-raw">
-        <th>Title</th>
-        <th>Author</th>
-        <th>ID</th>
-        <th></th>
-      </tr>
-    </thead>
-    <tbody id="book-list" class="bookList"></tbody>
-  </table> <br> <br>
-  <hr>
-  <br>
-  <form action="" id="book-form">
-    <div>
-      <label for="title"></label>
-      <input type="text" id="title" placeholder="Tittle" class="form-control">
+  <nav class="nav-bar navbar border">
+    <span>Awesome Books</span>
+    <ul class="ulNav">
+      <li class="border-right">
+        <a onclick="show(s1)">List |</a>
+      </li>
+      <li class="border-right">
+        <a onclick="show(s2)">Add Book |</a>
+      </li>
+      <li>
+        <a onclick="show(s3)">Contact</a>
+      </li>
+    </ul>
+  </nav>
+  <div class="timeDiv">
+    <span id="currentTime"></span>
+  </div>
+  <section class="books-list visible" id="s1">
+    <h1>Awesome books</h1>
+    <table class="table table-striped">
+      <thead>
+        <tr class="t-raw">
+          <th>Title</th>
+          <th>Author</th>
+          <th>ID</th>
+          <th></th>
+        </tr>
+      </thead>
+      <tbody id="book-list" class="bookList"></tbody>
+    </table>
+  </section>
+  <section class="addBook-section invisible" id="s2">
+    <h2>Add a new book</h2>
+    <form action="" id="book-form">
+      <div>
+        <label for="title"></label><br>
+        <input type="text" id="title" placeholder="Tittle" class="form-control"><br>
+      </div>
+      <div>
+        <label for="author"></label><br>
+        <input type="text" id="author" placeholder="Author" class="form-control">
+      </div>
+      <div>
+        <label for="isbn"></label><br>
+        <input type="text" id="isbn" placeholder="ID" class="form-control">
+      </div>
```

Comment:
- [ ] We should try to stick as much to the wireframe provided to us. There is no need to take the `ID` as input from the user. It should be assigned automatically when a user creates a list. Kindly try to fix it.

Yours
![image](https://user-images.githubusercontent.com/53019916/130230165-dda1ae24-f5dc-41f3-a6ee-475fae60ccdb.png)

Original
![image](https://user-images.githubusercontent.com/53019916/130230218-4db95814-31dd-478a-85e2-aa5ce22da4b2.png)

Reply:
Resolved